app/server.js

```diff
@@ -84,4 +87,14 @@ async function createServer () {
   return server
 }
 
+function loggingDecorator (request) {
```

Comment:
Yep feel this might be better all in one plugin.

Reply:
Will consider this for a future improvement, maybe next time we change

Reply:
Did you see my earlier comment though? https://github.com/DEFRA/ffc-ahwr-farmer-claim/pull/418#discussion_r1907049226

Reply:
See above, have refactored it a bit